ajax/libs/bootstrap-combobox/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "bootstrap-combobox",
+  "filename": "js/bootstrap-combobox.min.js",
+  "description": "A combobox plugin that works with twitter bootstrap",
+  "version": "1.1.7",
+  "author": {
+    "name": "Daniel Farrell",
+    "url": "https://github.com/danielfarrell"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/danielfarrell/bootstrap-combobox"
+  },
+  "keywords": [
+    "bootstrap-combobox",
+    "bootstrap",
+    "combobox"
+  ],
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/danielfarrell/bootstrap-combobox.git",
+    "basePath": "",
+    "files": [
+      "+(js|css)!(tests)/bootstrap-combobox*"
```

Comment:
This pattern didn't grab the `tests` folder because of you put `tests` in the wrong please but not because of you exclude it, before `/` is the name of first level of directory, can you try it again?